api-scanner/src/test/java/net/corda/plugins/AnnotatedInterfaceTest.java

```diff
@@ -44,9 +44,13 @@ public void testAnnotatedInterface() throws IOException {
 
         Path api = pathOf(testProjectDir, "build", "api", "annotated-interface.txt");
         assertThat(api).isRegularFile();
-        assertThat(Files.readAllLines(api)).containsOnlyOnce(
-            "@net.corda.annotation.AlsoInherited @net.corda.annotation.IsInherited @net.corda.annotation.NotInherited public interface net.corda.example.HasInheritedAnnotation",
-            "@net.corda.annotation.AlsoInherited @net.corda.annotation.IsInherited public interface net.corda.example.InheritingAnnotations extends net.corda.example.HasInheritedAnnotation"
-        );
+        assertEquals(
+            "@AlsoInherited\n" +
+            "@IsInherited\n" +
+            "@NotInherited\n" +
+            "public interface net.corda.example.HasInheritedAnnotation\n" +
+            "##\n" +
+            "public interface net.corda.example.InheritingAnnotations extends net.corda.example.HasInheritedAnnotation\n" +
```

Comment:
The _whole point_ of this test is that interface `InheritingAnnotations` should inherit `@IsInherited` and `@AlsoInherited`. Did you notice that you'd changed the expectation here? See the above comment that interfaces should be using `readInterfaceAnnotationsFor()`.